xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java

```diff
@@ -64,10 +64,10 @@ private GeminiAdapters() {}
     return currencyPairs;
   }
 
-  public static CurrencyPair adaptCurrencyPair(String GeminiSymbol) {
+  public static CurrencyPair adaptCurrencyPair(String symbol) {
 
-    String tradableIdentifier = GeminiSymbol.substring(0, 3).toUpperCase();
-    String transactionCurrency = GeminiSymbol.substring(3).toUpperCase();
+    String tradableIdentifier = symbol.substring(0, symbol.lenghth() - 3).toUpperCase();
+    String transactionCurrency = symbol.substring(symbol.lenghth() - 3).toUpperCase();
```

Comment:
is this `length()` spelling compiling?

Reply:
also would a 4 length transaction currency break this?

Reply:
looks like the other `length()` is still misspelled, is your IDE not highlighting this as a compilation error?